clients/python/README.md

```diff
@@ -56,11 +56,40 @@ Owner and controller of cells in the game. As a programmer of an AI, you are a
             and a dumb AI will take over the player's cells until the player
             becomes active once again.
 
+### Cell
```

Comment:
But since it is a competition, if they don't read the documentation, it is a bit their loss no? I don't mind either way, but I don't think we should baby the participants no?